src/main.rs

```diff
@@ -50,7 +50,13 @@ fn main() {
 
     logger::configure(env::var_os(vars::EXA_DEBUG));
 
-    let args: Vec<_> = env::args_os().skip(1).collect();
+    let cli_args = env::args_os().skip(1);
+    let args = get_args_from_env_var()
```

Comment:
I'm not sure this is the most performant way, tried to use `append` or `extend` but no luck 😅